debian/postinst

```diff
@@ -19,20 +22,28 @@ export_gpg_key_from_shared_keyring() {
     fi
 }
 
+unconfigure_esm() {
+    rm -f $ESM_APT_GPG_KEY $ESM_APT_GPG_KEY_OLD $ESM_APT_SOURCE_FILE
+    rm -f $ESM_APT_PREF_FILE
+}
+
 configure_esm() {
     rm -f $ESM_APT_GPG_KEY_OLD  # Remove retired key from key list
     if [ ! -f "$ESM_APT_GPG_KEY" ]; then
         export_gpg_key_from_shared_keyring $ESM_KEY_ID_TRUSTY
     fi
 
+    if [ -e "$ESM_APT_SOURCE_FILE_OLD" ]; then
+	mv $ESM_APT_SOURCE_FILE_OLD $ESM_APT_SOURCE_FILE
+    fi
```

Comment:
You also need to update `debian/postrm` to remove the new name instead of the old one.

Reply:
thanks added that for the purge